base_country_state_translatable/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
in v11, coding line is not necessary.
In general in `__init__.py`, no need copyright/license